### PR TITLE
Log siret behaviour to investigate Sentry #2371

### PR DESCRIPTION
**Pull request by mmagn**
#shame 😞

### CODE REVIEW

File: app/controllers/users/dossiers_controller.rb [hunk 1]
```diff
@@ -106,13 +106,16 @@ def siret_informations
     update_current_user_siret!(siret)
 
     etablissement_attributes = SIRETService.fetch(siret, @facade.dossier)
+    Rails.logger.info("etablissement_attributes for siret #{siret}, #{etablissement_attributes.present?}")
 
     if etablissement_attributes.present?
       etablissement_attributes = ActionController::Parameters.new(etablissement_attributes).permit!
       etablissement = @facade.dossier.create_etablissement(etablissement_attributes)
       if etablissement.save
+        Rails.logger.info("etablissement saved, #{etablissement.id} , #{siret}")
```
Comment (gregoirenovel) [inline app/controllers/users/dossiers_controller.rb]: Pas assez clair :
`Rails.logger.info("etablissement saved, id: #{etablissement.id} , siret: #{siret}")`

File: app/controllers/users/dossiers_controller.rb [hunk 2]
```diff
@@ -106,13 +106,16 @@ def siret_informations
     update_current_user_siret!(siret)
 
     etablissement_attributes = SIRETService.fetch(siret, @facade.dossier)
+    Rails.logger.info("etablissement_attributes for siret #{siret}, #{etablissement_attributes.present?}")
```
Comment (gregoirenovel) [inline app/controllers/users/dossiers_controller.rb]: ` +    Rails.logger.info("etablissement_attributes for siret #{siret} present? #{etablissement_attributes.present?}") `

File: app/controllers/users/dossiers_controller.rb [hunk 3]
```diff
@@ -106,13 +106,16 @@ def siret_informations
...
 
     if etablissement_attributes.present?
       etablissement_attributes = ActionController::Parameters.new(etablissement_attributes).permit!
       etablissement = @facade.dossier.create_etablissement(etablissement_attributes)
       if etablissement.save
+        Rails.logger.info("etablissement saved, #{etablissement.id} , #{siret}")
         @facade.dossier.mandataire_social!(current_user.france_connect_information)
       else
+        Rails.logger.info("etablissement not saved, #{etablissement.errors.full_messages} , #{siret}")
```
Comment (gregoirenovel) [inline app/controllers/users/dossiers_controller.rb]: `Rails.logger.info("etablissement not saved, siret: #{siret}, errors: #{etablissement.errors.full_messages}") `

**Comment by mmagn** [comment]
up

**Pull request merged**